### PR TITLE
[Stats Refresh] initial Post Stats view

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref #11189 

This is the first step in creating the Post Stats view. It displays a table view with the post title card, with the post title from the selected post.

To test:

---
- On Insights `Latest Post Summary`, select `View more`.
- Verify the Post Stats view is displayed with the post title.

lps

---
- On Period `Posts and Pages`, select a row.
- Verify the Post Stats view is displayed with the post title from the selected row.

pandp

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - 

---
ipad_port

---
ipad_land

**Comment by SylvesterWilmott** [comment]
Looks good @ScoutHarris !

A couple of things, can we just add a 2pt space between the two labels, the vertical rhythm seems a little off when the post title wraps onto a new line.

Screenshot 2019-03-11 at 08 56 10

Also can we use the typeface Noto Serif for the post title

File: WordPress/Classes/ViewRelated/Stats/Insights/Latest Post Summary/LatestPostSummaryCell.swift [hunk 1]
```diff
@@ -146,13 +147,13 @@ private extension LatestPostSummaryCell {
         }
 
         let postAge = lastPostInsight?.publishedDate.relativeStringInPast() ?? ""
-        let postTitle = lastPostInsight?.title.nonEmptyString() ?? NSLocalizedString("(No Title)", comment: "Empty Post Title")
+        postTitle = lastPostInsight?.title.nonEmptyString() ?? NSLocalizedString("(No Title)", comment: "Empty Post Title")
```
Comment (stevebaranski) [inline WordPress/Classes/ViewRelated/Stats/Insights/Latest Post Summary/LatestPostSummaryCell.swift]: It's unfortunate that the optionality propagates to lines 152 & 156.

If we change line 33 to:
```
private(set) var postTitle = NSLocalizedString("(No Title)", comment: "Empty Post Title")
```

then this segment might be simplified to something like:
```
if let title = lastPostInsight?.title, !title.isEmpty {
    postTitle = title
}
```

And we can remove the `!` on the aforementioned lines.

Thoughts?

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - 

> can we just add a 2pt space between the two labels

Done!

> can we use the typeface Noto Serif for the post title

Gah! Sorry I missed that. Done as well.

What do you think?

title_card

**Comment by ScoutHarris** [comment]
@stevebaranski I've made all requisite changes, if you care to take another look. Thanks!

**Comment by ScoutHarris** [comment]
@SylvesterWilmott  - I'm going to merge this. If any additional changes are required just let me know and I can make them in a future PR.

**Pull request merged**